lisa/notifiers/log_file.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import logging
+from dataclasses import dataclass
+from typing import Any, List, Type, cast
+
+from dataclasses_json import dataclass_json
+
+from lisa import notifier, schema
+from lisa.environment import Environment
+from lisa.testsuite import TestResultMessage
+from lisa.util import constants, dump_file
+
+
+@dataclass_json()
+@dataclass
+class LogFileSchema(schema.TypedSchema):
+    log_level: str = logging.getLevelName(logging.DEBUG)
+
+
+class LogFile(notifier.Notifier):
+    """
+    It's a sample notifier, output subscribed message to a log file.
```

Comment:
It needs to specify the reason that we create this notifier. It's similar with the PR description.